Test/Precompiled/Catch2/CMakeLists.txt

```diff
@@ -14,4 +14,9 @@ set_target_properties(${TARGET_TEST} PROPERTIES FOLDER ${FOLDER})
 target_link_libraries(${TARGET_TEST} ${TARGET_MAIN})
 
 target_compile_definitions(${TARGET_TEST} PRIVATE CATCH_CONFIG_FAST_COMPILE)
-target_precompile_headers(${TARGET_TEST} PRIVATE ${CONAN_CATCH2_ROOT}/include/catch2/catch.hpp)
\ No newline at end of file
+
+# TODO This is definately bad, gotta rework this ASAP
```

Comment:
Replace with @todo so doxygen can see this bookmark?

Reply:
I don't think it parses CMakelists.